Command/IndexRebuildCommand.php

```diff
@@ -14,12 +14,18 @@
 use Massive\Bundle\SearchBundle\Search\Event\IndexRebuildEvent;
 use Massive\Bundle\SearchBundle\Search\SearchEvents;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\FormatterHelper;
+use Symfony\Component\Console\Helper\QuestionHelper;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
 
 /**
  * Comand to build (or rebuild) the search index.
+ *
+ * TODO: Rename this to ReIndexCommand
```

Comment:
How about renaming it now, and creating this class again, decorating the renamed class, and mark it as deprecated?